### PR TITLE
RELATED: RAIL-2999 - Tiger backend & catalog export fixes

### PR DESCRIPTION
**Pull request by lupko**
We had a lot of code which needed to load all entities from server. However none of this code was actually using the paging. It was always just getting one (huge) page. If there were more results, they would not be included.

This PR fixes all occurrences of this so that the MetadataUtilities.getAllPagesOf is used to load everything. This had to be done in sdk-backend-tiger and in catalog-export tiger loader.

Additionally, the PR brings in a couple of trivial refactors for consistency. An instance of ITigerClient is now everywhere stored in variable/field/prop `client`. We had `tigerClient` sometimes and sometimes `sdk`. Also updated the catalog-export so that the different tiger loader methods always start with the client as first argument.

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2813/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4439/ : SUCCESS in 13m 50s

File: libs/sdk-backend-tiger/src/backend/organization/index.ts [hunk 1]
```diff
@@ -46,8 +47,9 @@ export class TigerOrganizations implements IOrganizations {
 
     public async getCurrentOrganization(): Promise<IOrganization> {
         // TODO: replace with direct call of TigerClient (once methods are generated from OpenAPI)
-        const organizationData = (await this.authCall((sdk) => sdk.axios.get("/api/entities/organization")))
-            .data;
+        const organizationData = (
+            await this.authCall((client) => client.axios.get("/api/entities/organization"))
```
Comment (kandl) [inline libs/sdk-backend-tiger/src/backend/organization/index.ts:51]: Now it's generated on tiger client (we can remove TODO)
`client.organizationObjects.getAllEntitiesOrganizations(...);`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2820/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4444/ : SUCCESS in 13m 39s

File: libs/sdk-backend-tiger/src/backend/organization/index.ts (hunk 1 above)
Comment (lupko) [reply libs/sdk-backend-tiger/src/backend/organization/index.ts:51]: I propose to do it as part of followup after your PR. the org utilities getAllPagesOf will be needed here anyway. 

what do you think?

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1124

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/827/ : SUCCESS in 25s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1091/ : SUCCESS in 29s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4445/ : SUCCESS in 13m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/198/ : SUCCESS in 13m 41s (non-voting)

**Pull request merged**